packages/publikator/package.json

```diff
@@ -1,11 +1,20 @@
 {
-  "name": "publikator-backend",
+  "name": "@orbiting/backend-modules-publikator",
   "version": "0.0.1",
-  "engines": {
-    "node": "14.x"
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/orbiting/backends.git"
   },
-  "private": true,
```

Comment:
Why isn't this a `private` repo? Do we plan to publish it to `npm`?

https://docs.npmjs.com/files/package.json#private

Reply:
No. But as I understand it, `private` is there to ensure that a package is never published to a registry and we don't need that from a securit standpoint.